test/functional/test_framework/blocktools.py

```diff
@@ -87,6 +87,14 @@ def create_block(hashprev=None, coinbase=None, ntime=None, *, version=None, tmpl
     block.calc_sha256()
     return block
 
+def create_large_block(hashprev=None, ntime=None, height=None, scriptPubKey=CScript([OP_RETURN] + [OP_TRUE] * 950000)):
+    coinbase_tx = create_coinbase(height)
+    coinbase_tx.vin[0].nSequence = 2 ** 32 - 1
```

Comment:
I wonder if there is reason to change the sequence?

Reply:
I can make it a parameter if needed.

Reply:
I removed it in #24865, thanks.